gemini/prompts/prompt_optimizer/vertex_ai_prompt_optimizer_ui.ipynb

```diff
@@ -0,0 +1,907 @@
+{
```

Comment:
<p>Should it be PROMPT_TEMPLATE according to what we discussed? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1099/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Changed here, but the flag name change on the binary is not released yet, so below in params we are still using the old names, I'll fix those after the release.</p>